docs/using-the-compiler.rst

```diff
@@ -301,3 +301,13 @@ Error types
 11. ``CompilerError``: Invalid use of the compiler stack - this should be reported as an issue.
 12. ``FatalError``: Fatal error not processed correctly - this should be reported as an issue.
 13. ``Warning``: A warning, which didn't stop the compilation, but should be addressed if possible.
+
+
+
+Decompiler
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There is no decompiler for Solidity, but `Porosity <https://github.com/comaeio/porosity>`_ is close.
```

Comment:
The github page now mentions that it is unmaintained. Perhaps we should not link it anymore?